### PR TITLE
XML file links, exception handling, better img integration support.

### PR DESCRIPTION
**Pull request by phanicode**

### CODE REVIEW

File: PyWhatsapp.py [hunk 1]
```diff
@@ -198,34 +206,64 @@ def send_attachment():
...
+            traceback.print_exc()
+    else:
+        try: 
+            # IMG attatchment button
+            docButton = browser.find_element_by_xpath('//*[@id="main"]/footer/div[1]/div[1]/div[2]/div/span/div/div/ul/li[1]/button')
+            docButton.click()
+        except:
+            # Check for traceback errors with XML imports
+            traceback.print_exc()
+
```
Comment (shauryauppal) [inline PyWhatsapp.py:247]: Why so much spacing in code here?

File: PyWhatsapp.py [hunk 2]
```diff
@@ -198,34 +206,64 @@ def send_attachment():
...
-
     docPath = os.getcwd() + "\\Documents\\" + doc_filename
-
-    autoit.control_focus("Open", "Edit1")
+    
+    try:
+        autoit.control_focus("Open", "Edit1")
+    except :
+        traceback.print_exc()
+
```
Comment (shauryauppal) [inline PyWhatsapp.py:257]: Here also can we remove spacing or keep it to one line

File: PyWhatsapp.py [hunk 3]
```diff
@@ -198,34 +206,64 @@ def send_attachment():
...
-    clipButton = browser.find_element_by_xpath('//*[@id="main"]/header/div[3]/div/div[2]/div/span')
+    clipButton = browser.find_element_by_xpath('//*[@id="main"]/footer/div[1]/div[1]/div[2]/div/div/span')
     clipButton.click()
+    
     time.sleep(1)
-
     # To send a Document(PDF, Word file, PPT)
-    docButton = browser.find_element_by_xpath('//*[@id="main"]/header/div[3]/div/div[2]/span/div/div/ul/li[3]/button')
-    docButton.click()
+    # modified code to work with my particular implementation
```
Comment (shauryauppal) [inline PyWhatsapp.py:227]: Cool got it can we remove these comments, don't want this to go to production.

File: PyWhatsapp.py [hunk 4]
```diff
@@ -275,12 +313,14 @@ def sender():
                 try:
                     send_attachment()
                 except:
-                    print('Attachment not sent.')
+                    # No need to print File not sent as try except statements are in function.
```
Comment (shauryauppal) [inline PyWhatsapp.py:316]: Please remove the comment here and below

**Review by shauryauppal — CHANGES_REQUESTED** [review]
Request some minor changes. The rest looks good. Thanks for contributing to open source. :-)

**Review by shauryauppal — APPROVED** [review]

**Comment by shauryauppal** [comment]
Congrats @phanicode on officially being part of PyWhatsapp as a Contributor.

**Comment by khale11** [comment]
Hi مرحبا

**Pull request merged**